clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1861,13 +1867,23 @@ def CallOp : CIR_CallOpBase<"call", [NoRegionArguments]> {
   let arguments = commonArgs;
 
   let builders = [OpBuilder<(ins "mlir::SymbolRefAttr":$callee,
-                                 "mlir::Type":$resType,
-                                 "mlir::ValueRange":$operands), [{
+                                "mlir::Type":$resType,
+                                "mlir::ValueRange":$operands),
+                            [{
       $_state.addOperands(operands);
       $_state.addAttribute("callee", callee);
       if (resType && !isa<VoidType>(resType))
         $_state.addTypes(resType);
-    }]>];
+    }]>,
+                  OpBuilder<(ins "mlir::Value":$callee, "mlir::Type":$resType,
```

Comment:
The formatting here is off. Can you align the two OpBuilder lines and indent everything else relative to them?

Reply:
Updated.